packages/engine/apache-arrow-js-bundle/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "apache-arrow-js-bundle",
```

Comment:
I chose not to do that because I thought our naming conventions were to do with publishing the package. But yeah, can see that was incorrect. Will update :)